ProjFS.Mac/PrjFSKext/public/PrjFSXattrs.h

```diff
@@ -30,3 +31,10 @@ struct PrjFSFileXAttrData
     unsigned char providerId[PrjFS_PlaceholderIdLength];
     unsigned char contentId[PrjFS_PlaceholderIdLength];
 };
+
+struct PrjFSDirectoryXAttrData
+{
+    PrjFSXattrHeader header;
+
+    unsigned char providerId[PrjFS_PlaceholderIdLength];
```

Comment:
Do we actually do anything with this providerId or is it basically metadata on the directory?

Reply:
[@wilbaker's comment in the issue this is supposed to resolve](https://github.com/microsoft/VFSForGit/issues/515#issuecomment-524957066) explains it better than I ever could. In short, we don't really do anything with it at the moment.